ocs_ci/ocs/constants.py

```
@@ -97,6 +97,31 @@
 ROOK_CSI_CEPHFS_DIR = os.path.join(ROOK_EXAMPLES_DIR, "csi", "cephfs")
 CLEANUP_YAML = "cleanup.yaml.j2"
 MANIFESTS_DIR = "manifests"
+VOLSYNC_HELM_CONFIG = os.path.join(TOP_DIR, MANIFESTS_DIR, "volsync-helm.yaml")
+MIGRATE_PVC_PV_SCRIPT = os.path.join(TOP_DIR, MANIFESTS_DIR, "migrate-pvc-pv.sh")
+MINIO_NAMESPACE = "minio"
+MINIO_YAML = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "minio.yaml")
+RECIPE_CRD_YAML = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "recipe-crd.yaml")
+MINIO_INTERNAL_ENDPOINT = "http://minio.minio.svc:9000"
+MINIO_BUCKET = "bucket"
+MINIO_ACCESS_KEY = "minio"
+MINIO_SECRET_KEY = "minio123"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "MINIO_SECRET_KEY"

(S105)